pyproject.toml

```diff
@@ -44,6 +44,7 @@ test = ["pytest", "pytest-cov", "pytest-mpl", "matplotlib"]
 lint = ["black", "isort>=6.0.0", "pylint", "mypy", "qbraid-cli>=0.10.2"]
 docs = ["sphinx>=7.3.7,<8.3.0", "sphinx-autodoc-typehints>=1.24,<3.2", "sphinx-rtd-theme>=2.0.0,<4.0.0", "docutils<0.22", "sphinx-copybutton"]
 visualization = ["matplotlib"]
+pulse = ["openpulse[parser]>=1.0.1"]
```

Comment:
This looks good, can you also add a test to make sure that whenever we install `pyqasm` with the `pulse` extra, the `openpulse` library can be correctly imported? It will be good to create a `tests/pulse` directory which will eventually contain all the tests. Something like `test_import.py` should suffice for now.